bundles/org.openhab.binding.cul.max/README.md

```diff
@@ -0,0 +1,204 @@
+# MAX!CUL Binding
+
+The aim of this binding is to allow the connection from openHAB to [eQ-3 MAX! Home Solution](http://www.eq-3.de/) devices (wall thermostat/radiator valves)
+using the [CUL USB dongle](http://busware.de/tiki-index.php?page=CUL) rather than the MAX!Cube.
+This should allow greater control over the devices than the cube offers as all interaction is handled manually.
+
+## Supported Things
+
+This binding support 6 different things types
+
+| Thing          | Type   | Description                                                                                                        |
+|----------------|--------|--------------------------------------------------------------------------------------------------------------------|
+| cul_max_bridge | Bridge | CUL if using a serial CUL
+| cun_max_bridge | Bridge | CUN if using a network CUL
+| maxcul_bridge  | Bridge | This is the MAX! Cube emulation for the CUL.                                                                                 |
+| thermostat     | Thing  | This is for the MAX! Heating Thermostat. This is also used for the power plug switch "Zwischenstecker-Schaltaktor". |
+| thermostatplus | Thing  | This is for the MAX! Heating Thermostat+. This is the type that can hold the program by itself.                    |
+| wallthermostat | Thing  | MAX! Wall Thermostat.                                                                                              |
+| ecoswitch      | Thing  | MAX! Ecoswitch.                                                                                                    |
+| shuttercontact | Thing  | MAX! Shuttercontact / Window Contact.                                                                              |
+
+Generally one does not have to worry about the thing types as they are automatically defined.
+If for any reason you need to manually define the Things and you are not exactly sure what type of thermostat you have, you can choose `thermostat` for both the thermostat and thermostat+, this will not affect their working.
+
+## Discovery
+
+Start pairing at your MAX! device. It will show up in the inbox and you can it to your configuration. Serial number and rfAddress is set already
+After adding too your configuration you have to set channel pair_mode to ON on your maxcul_bridge and restart
+the pairing on your MAX! device to start the pairing sequence. 
+If the device is already paired and starts re-pairing e.g. after battery change, the state of channel pair_mode is ignored.

```

Comment:
You could add a reference to the pairing chapter: `[Pairing](#pairing)`

Reply:
done.